op-chain-ops/cmd/op-upgrade/main.go

```diff
@@ -99,15 +99,10 @@ func entrypoint(ctx *cli.Context) error {
 	batch := safe.Batch{}
 
 	for _, chainConfig := range targets {
-		name, err := toDeployConfigName(chainConfig)
-		if err != nil {
-			log.Warn("Skipping unsupported network", "name", chainConfig.Name)
-			continue
-		}
-
+		name, _ := toDeployConfigName(chainConfig)
 		config, err := genesis.NewDeployConfigWithNetwork(name, deployConfig)
 		if err != nil {
-			return err
+			log.Warn("Cannot find deploy config for network", "name", chainConfig.Name, "deploy-config-name", name, "path", deployConfig)
 		}
```

Comment:
Should we also be performing a deploy config check here following the `check-deploy-config` op-chain-ops [command](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/cmd/check-deploy-config/main.go#L47-L50)?

https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/cmd/check-deploy-config/main.go#L47-L50

Reply:
It is a good idea to call check, thanks for the idea. I feel like this solves your other comment too

Reply:
Yep agreed, re-reviewing... 😄